lib/web_ui/lib/src/engine/initialization.dart

```diff
@@ -226,6 +226,7 @@ Future<void> initializeEngineUi() async {
 
   RawKeyboard.initialize(onMacOs: operatingSystem == OperatingSystem.macOs);
   KeyboardBinding.initInstance();
+  ViewFocusBinding.initialize();
```

Comment:
🌶️ 👍

Reply:
(There's a better place for the init/dispose though!)

Reply:
:handshake: